Scripts/Editor/AssemblyInjections/UnityEditor/Unity.InternalAPIEditorBridge.001.asmdef

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Unity.InternalAPIEngineBridge.001",
```

Comment:
hah, wow ok. More magic Unity names...
Cool ability though!

Reply:
Yeah odd way to do it, but after many a google search and travelling through the forums.... I finally found it